lib/features/steps/request_assertion_steps.rb

```diff
@@ -13,29 +15,35 @@
   step 'I wait to receive 1 request'
 end
 
-# Continually checks to see if the required amount of requests have been received. Times out after 30 seconds.
+# Continually checks to see if the required amount of requests have been received.
+# Times out according to @see MazeRunner.config.receive_requests_wait.
 #
 # @step_input request_count [Integer] The amount of requests expected
 Then('I wait to receive {int} request(s)') do |request_count|
-  max_attempts = 300
+  INTERVAL = 0.1
+  max_attempts = MazeRunner.config.receive_requests_wait / INTERVAL
   attempts = 0
   received = false
   until (attempts >= max_attempts) || received
     attempts += 1
     received = (Server.stored_requests.size >= request_count)
-    sleep 0.1
+    sleep INTERVAL
   end
   unless received
     raise "Expected #{request_count} requests but received #{Server.stored_requests.size} within the 30s timeout. " \
-      'This could indicate that Bugsnag crashed with a fatal error, or that it hasn’t made the requests that it ' \
-      'should have done. Please check the device logs to confirm.'
+      "This could indicate that:\n" \
+      " - Bugsnag crashed with a fatal error\n" \
+      " - Bugsnag did not make the requests that it should have done.\n" \
+      " - The requests were made, but not deemed to be valid (e.g. missing integrity header)\n" \
+      'Please check the Maze Runner and device logs to confirm.'
```

Comment:
There's multiline strings:

```rb
    raise %Q(Expected #{request_count} requests but received #{Server.stored_requests.size} within the 30s timeout.
      This could indicate that:
      - Bugsnag crashed with a fatal error
      - Bugsnag did not make the requests that it should have done.
      - The requests were made, but not deemed to be valid (e.g. missing integrity header)
      Please check the Maze Runner and device logs to confirm.
    )
```

You can also use [heredoc](https://en.wikipedia.org/wiki/Here_document):

```rb
    raise <<-MESSAGE
    Expected #{request_count} requests but received #{Server.stored_requests.size} within the 30s timeout.
      This could indicate that:
      - Bugsnag crashed with a fatal error
      - Bugsnag did not make the requests that it should have done.
      - The requests were made, but not deemed to be valid (e.g. missing integrity header)
      Please check the Maze Runner and device logs to confirm.
    MESSAGE
```

[More examples in Ruby](https://www.rubyguides.com/2018/11/ruby-heredoc/)

Reply:
Thanks.  I hate myself for having done it that way.